### PR TITLE
Hide and Seek Game

### LINKED ISSUE #4609: [New game]: Hide and Seek
### 🎮 Game Request

Hide and Seek will be a fun and interesting game. The game will feature a intuitive 4x4 (rows and columns) square tiles. In the square tiles there are certain set of emojis present on it (having multiple two two pairs of them will be present). The square tiles will be hidden by default and it will show it's content for 4 seconds when the user clicks the tile to find a pair of same emoji. If the user finds all the pairs of same emojis, he/she will win the game otherwise he/she lose the match.

### Point down the features

- Will help the user in memorising the things.
- Boosts user logical analysis and interpretation.
- Enhance the readability.

### Select program in which you are contributing

GSSoC24

### Code of Conduct

- [X] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) of this project.

### PR DESCRIPTION
**Pull request by PilotAxis**
## PR Description 📜

Hide and Seek is a memory testing game where the player has to flip the cards in order to find out the pairs of matched symbols. It enhances the player's cognitive thinking and it's memory skills. Hide and Seek is a fun and beautifully created to enjoy the game without any errors.
Fixes #4609 

<hr>
 
## Mark the task you have completed ✅

- [x] I follow [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md) & [CODE OF CONDUCT](https://github.com/kunjgit/GameZone/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I have followed proper naming convention showed in [CONTRIBUTING GUIDELINE](https://github.com/kunjgit/GameZone/blob/main/.github/CONTRIBUTING_GUIDELINE.md)
- [x] I have added screenshot for website preview in assets/images 
- [x] I have added entries for my game in main README.md
- [x] I have added README.md in my folder 
- [x] I have added working video of the game in README.md (optional)
- [x] I have specified the respective issue number for which I have requested the new game.

<hr>

## Add your screenshots(Optional) 📸

beginning

--- 
<br>

## Thank you soo much for contributing to our repository 💗

### CODE REVIEW

**Comment by kunjgit** [comment]
Thank you @PilotAxis ,for creating the PR and contributing to our GameZone 💗 
 Review team will review the PR and will reach out to you soon! 😇 
 Make sure that you have marked all the tasks that you are done with ✅. 
 Thank you for your patience! 😀

**Comment by PilotAxis** [comment]
@ishita-43 Please review my PR...

**Comment by PilotAxis** [comment]
@ishita-43 I have the game image in the assets folder of GameZone. Please review my PR.

File: README.md [hunk 1]
```diff
@@ -286,6 +286,7 @@ This repository also provides one such platforms where contributers come over an
 | [QuickFingers](https://github.com/kunjgit/GameZone/tree/main/Games/QuickFingers)       | 
 | [Physics_Quizz](https://github.com/kunjgit/GameZone/tree/main/Games/Physics_Quizz)     |
 | [Tiny_Fishing](https://github.com/kunjgit/GameZone/tree/main/Games/Tiny_Fishing)       | 
+| [Hide and Seek](https://github.com/PilotAxis/GameZone/tree/main/Games/Hide%20and%20Seek)    |
```
Comment (ishita-43) [inline README.md:289]: remove this duplicate entry

File: README.md [hunk 2]
```diff
@@ -348,8 +349,12 @@ This repository also provides one such platforms where contributers come over an
 | [Intellect Quest](https://github.com/Will2Jacks/GameZoneForked/tree/Task/Games/Intellect_Quest) |
 | [Number_Guessing_Game](https://github.com/kunjgit/GameZone/tree/main/Games/Number_Guessing_Game) |
 | [Modulo_Game](https://github.com/kunjgit/GameZone/tree/main/Games/Modulo_Game) |
+
+| [Hide_and_Seek](https://github.com/PilotAxis/GameZone/tree/main/Games/Hide_And_Seek) |
```
Comment (ishita-43) [inline README.md:353]: remove the duplicate entry

**Review by ishita-43 — CHANGES_REQUESTED** [review]

File: README.md (hunk 1 above)
Comment (PilotAxis) [reply README.md]: Removed the duplicate entry, Kindly proceed with the PR.

File: README.md (hunk 2 above)
Comment (PilotAxis) [reply README.md]: Removed the duplicate entry, Kindly proceed with the PR

**Comment by PilotAxis** [comment]
@ishita-43 Resolved the duplicate entries issue, kindly proceed with the PR...

**Comment by ishita-43** [comment]
Try to resolve merge conflicts or sync the readme.md file with latest changes and just add your game entry in there

**Review by ishita-43 — APPROVED** [review]
Good to go!!

**Comment by kunjgit** [comment]
Thank you  @PilotAxis , for your valuable time and contribution in our GameZone 💗. 
 It’s our GameZone, so Let’s build this GameZone altogether !!🤝
  Hoping to see you soon with another PR again 😇 
 Wishing you all the best for your journey into Open Source🚀

**Comment by PilotAxis** [comment]
@ishita-43 In the issue the label was assigned with the level of 3 but here you had added level 2.

**Pull request merged**